packages/exporter-trace-otlp-grpc/README.md

```diff
@@ -107,6 +107,29 @@ provider.register();
 
 Note, that this will only work if TLS is also configured on the server.
 
+## Exporter Timeout Configuration
+
+The OTLPTraceExporter has a timeout configuration option which is the maximum time, in milliseconds, the OTLP exporter will wait for each batch export. The default value is 10000ms.
```

Comment:
discussion:
I wonder if this time should be set in seconds or ms.
ms feels more native to JS.
On the other end, the spec states the default value as `10s` and ruby and python use seconds for this value, and there is also a value in consistency across otel SDKs.
WDYT?

Reply:
I see what you mean. It's interesting that timeout for the BSP is [using ms](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md) and timeout for the exporter mentions seconds. 

I do like the consistency across other otel SDKs so I'm good with changing this to use seconds.

Reply:
> It's interesting that timeout for the BSP is [using ms](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md) and timeout for the exporter mentions seconds.

Interesting. I wonder if there is any thinking behind this or it was just added at different times and there was no notice on unifying it.

> I do like the consistency across other otel SDKs so I'm good with changing this to use seconds.

I now checked the java implementation and it uses ms, so there is currently no consistency with popular otel SDKs. I am good with keeping it as ms or changing it to seconds. Whichever one you think is better. Sorry for the trouble.

Reply:
No worries. I will keep it as ms.